main.nf

```diff
@@ -1053,6 +1067,100 @@ process Register_T1 {
     """
 }
 
+labels_for_reg
+    .join(t1_for_freesurfer_reg)
+    .set{labels_mat_for_reg}
+
+process Register_Freesurfer {
+    cpus 1
+
+    input:
+    set sid, file(aparc), file(wmparc), file(t1), file(affine),
+        file(warp) from labels_mat_for_reg
```

Comment:
Also, I'm wondering is this `warp` file `output1InverseWarp` or `output1Warp`?
In the older repo [here](https://github.com/scilus/tractoflow-ABS/blob/master/main.nf#L957-L962), it seems to be using `output1Warp`, but here is seems only `output1InverseWarp` is in `labels_mat_for_reg`? Thanks.

Reply:
Thanks it is fixed now.